.github/workflows/integration-tests-release.yaml

```diff
@@ -0,0 +1,219 @@
+
+          # install kubectl
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+          # install kind
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.10.0/kind-linux-amd64
+          chmod +x ./kind
+          sudo install -o root -g root -m 0755 ./kind /usr/local/bin/kind
+
+          # install jq
+          sudo apt-get install jq -y
+          
+          # install Helm
+          ./scripts/install-helm.sh
+
+          # Update Helm dependencies
+          ./scripts/update-helm-deps.sh
+
+      - name: Write Secrets
+        run: |
+          printf "%s" '${{ secrets.MEDUSA_SECRET }}' > ~/medusa_secret.yaml
+
+      - name: Check if integration tests can run
+        env:
+          SCENARIO: ${{ matrix.scenario }}
+        run: |
+          if [[ ( -n "${{ secrets.MEDUSA_SECRET }}" && $SCENARIO == *"S3"* ) || $SCENARIO != *"S3"* ]];
+          # Medusa on S3 requires a secret. Skip integration tests if the secret is absent.
+          then
+            echo "IT_CAN_RUN=yes" >> $GITHUB_ENV
+          else
+            echo "IT_CAN_RUN=no" >> $GITHUB_ENV
+          fi
+
+      - name: Run Integration Tests
+        if: ${{ env.IT_CAN_RUN == 'yes' }}
+        env:
+          SCENARIO: ${{ matrix.scenario }}
+          K8SSANDRA_CASSANDRA_VERSION: ${{ matrix.cassandra_version }}
+        run: |
+          # Set a valid name for the artifacts in case of failure
+          echo "ARTIFACT_NAME=k8s-artifacts-${K8SSANDRA_CASSANDRA_VERSION}-$(echo $SCENARIO|tr '":/' '___')" >> $GITHUB_ENV
+          make kind-integ-test TESTS="$SCENARIO" GO_FLAGS="-v" CLUSTER_CLEANUP="success"
+
+      - name: Collect artifacts
+        if: ${{ failure() }}
+        env:
+          ARTIFACTS_DIR: artifacts
+        run: tests/integration/scripts/collect_artifacts.sh
+
+      - uses: actions/upload-artifact@v2
+        if: ${{ failure() }}
+        with:
+          name: k8s-artifacts-${{ env.ARTIFACT_NAME }}
+          path: artifacts/
+      
+      # Uncomment to run with SSH on failure
+      - name: Setup tmate session
+        if: "failure() && contains(github.event.head_commit.message, '[SSH]')"
+        uses: mxschmitt/action-tmate@v3
+        timeout-minutes: 10
+  
+  release:
+    if: |
+      ${{ (( github.event_name == 'release' && github.event.action == 'published' ) 
+        || ( github.event_name == 'push' && github.ref == 'refs/heads/main' )) }}
+    runs-on: ubuntu-latest
+    needs: integration-tests
+    env:
+      HELM_VERSION: 3.5.3
+      YQ_VERSION: v4.6.3
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Update PATH
+        run: |
+          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
+
+      - name: Install and validate tool versions
+        run: |
+          scripts/install-helm.sh $HELM_VERSION
```

Comment:
Yet another different way of installing helm? Why not the same as integration-tests? And same for yq. Why the definition of version here, but not previously? It could in theory cause different behaviour.

Reply:
I kept it the way it was done in the release workflow, but agree helm should be installed with the same version (we're already using the same script).

Here we're installing `yq`, but the integration tests are installing `jq` 😉  Actually, this could be a leftover because I'm not sure what we need `jq` for 🤔  I'll remove it and see if something breaks.